cloudpickle/cloudpickle.py

```diff
@@ -60,8 +60,11 @@
 import weakref
 import uuid
 import threading
+import typing
 from enum import Enum
 
+from . import generic_shims
```

Comment:
Those shims should not be necessary just to fix "Add support for pickling dynamic generics on 3.7+". They are needed for Python 3.5 and 3.6 only. If it's too cumbersome to split the two changes, feel free to keep on working on #318 only.

Reply:
Of course. I'll remove it from this PR, and keep it only in the other PR.